library/src/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -647,6 +746,21 @@ public void highlightDates(Collection<Date> dates) {
     setAdapter(adapter);
   }
 
+  /**
+   * Unhighlights all previously highlighted days.
+   * Also clears {@link #highlightedCells} and {@link #highlightedCals}.
+   */
+  public void unHighlightAllDates() {
```

Comment:
Cool. I didn't know about (or have forgotten about) that one!
On Jan 18, 2015 2:12 PM, "Daugilas" <notifications@github.com> wrote:

> In library/src/com/squareup/timessquare/CalendarPickerView.java
> <https://github.com/square/android-times-square/pull/171#discussion-diff-23137589>
> :
>
> > @@ -647,6 +746,21 @@ public void highlightDates(Collection<Date> dates) {
> >      setAdapter(adapter);
> >    }
> >
> > +  /**
> > +   * Unhighlights all previously highlighted days.
> > +   * Also clears {@link #highlightedCells} and {@link #highlightedCals}.
> > +   */
> > +  public void unHighlightAllDates() {
>
> Its different than selected. There is a method highlightDates(Collection<Date>
> dates) from long ago. And unHighlightAllDates() removes highlights.
> clearHighlightedDates() though makes sense, will change to that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/square/android-times-square/pull/171/files#r23137589>.
>